users/models.py

```diff
@@ -175,9 +177,12 @@ def get_projects_work_percentage(self, from_date: Optional[date] = None, to_date
         work_hours_sum = work_hours_per_project.aggregate(
             sum_work_hours_sum=Coalesce(Sum("work_hours_sum"), timezone.timedelta())
         )["sum_work_hours_sum"]
-
         return {
-            project: ((project.work_hours_sum / work_hours_sum) * 100 if work_hours_sum.total_seconds() > 0 else 0)
+            project: (
+                [str(project.work_hours_sum)[:-3], (project.work_hours_sum / work_hours_sum) * 100]
+                if work_hours_sum.total_seconds() > 0
+                else 0
```

Comment:
Please create from this code separate private method. Method name should tell what is going on in here